obc_rate_noise_plots/plot_recent_rates.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+import os
+
+import matplotlib
+import numpy as np
+
+if __name__ == "__main__":
+    matplotlib.use("Agg")
+import shutil
+from pathlib import Path
+
+import matplotlib.pyplot as plt
+from Chandra.Time import DateTime
+from kadi import events
+from Ska.engarchive import fetch
+from Ska.Matplotlib import plot_cxctime
```

Comment:
There might be a case for going through and moving to the non-namespace versions with this packaging, but I'm OK with limiting this PR to the repackaging and we can do tiny PRs to improve things (and this being a non-fsds package, that is not a big process).